src/main/java/com/uid2/shared/store/RotatingKeyStore.java

```diff
@@ -162,6 +162,12 @@ public EncryptionKey getMasterKey(Instant now) {
             return this.getActiveSiteKey(Const.Data.MasterKeySiteId, now);
         }
 
+        @Override
+        public EncryptionKey getRefreshKey(Instant now) {
+            EncryptionKey refreshKey = this.getActiveSiteKey(Const.Data.RefreshKeySiteId, now);
+            return refreshKey != null ? refreshKey : this.getActiveSiteKey(Const.Data.MasterKeySiteId, now);
```

Comment:
Why is this fallback needed? This can be confusing/dangerous if something goes wrong.

Reply:
This is for handling legacy key file that doesn't have refresh key. Removed as we'll coordinate deployment to ensure that doesn't happen.